WcaOnRails/db/migrate/20181226115357_add_senior_members_column_to_teams.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddSeniorMembersColumnToTeams < ActiveRecord::Migration[5.2]
+  def change
+    add_column :team_members, :team_senior_member, :boolean, default: false, null: false
```

Comment:
The `team_` prefix seems unnecessary to me, but I'm down to leave it here for consistency with `team_leader`.

Reply:
I agree. Do you want me to send in a PR to remove both?

Reply:
I don't think it's necessary. I also think we're going to revisit this stuff at some point when we think again about how to make delegates teams, so I'd rather wait until then, because I'm sure this will change then.

Reply:
Ok!